modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -8,18 +8,52 @@ This guide contains an overview of the changes in Canvas Kit v9. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
+- [Glossary](#glossary)
 - [Codemod](#codemod)
-- [Component Deprecations](#component-deprecations)
+- [Token Updates](#token-updates)
+  - [Depth](#depth)
+- [Component Removals](#component-removals)
   - [Drawer](#drawer)
   - [Layout and Column](#layout-and-column)
-  - [Stack](#stack-hstack-vstack)
+  - [Stack, HStack, and VStack](#stack-hstack-and-vstack)
 - [Component Updates](#component-updates)
-  - [Buttons](#buttons)
-	- [Toast](#toast)
+  - [Button](#button)
+  - [Toast](#toast)
 - [Utility Updates](#utility-updates)
   - [focusRing](#focusring)
-- [Token Updates](#token-updates)
-	- [Depth](#depth)
+
+## Glossary
+
+### Main
+
+Our Main package of Canvas Kit tokens, components, and utilities at `@workday/canvas-kit-react` have
+undergone a full design and a11y review and are approved for use in product.
+
+Breaking changes to code in Main will only occur during major version updates.
```

Comment:
I've added a glossary explaining what Main, Preview, and Labs are since those terms are used frequently throughout this guide. I cobbled the definitions together using content from the module [READMEs](https://github.com/Workday/canvas-kit/blob/master/modules/preview-react/README.md) and the [Canvas Glossary](https://canvas.workday.com/getting-started/canvas-glossary#canvas-kit-preview). Let me know you think!